core/domain/question_services.py

```diff
@@ -142,6 +142,26 @@ def get_questions_and_skill_descriptions_by_skill_ids(
     return questions, skill_descriptions, next_cursor
 
 
+def get_questions_by_skill_ids(question_count, skill_ids):
```

Comment:
Please rename throughout to either question_count_per_skill or total_question_count, since otherwise it's unclear whether this represents a per-skill question count or a total question count. (Ditto up and down the stack.)

Reply:
Done.